DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: gghdx
 Title: HDX Theme, Scales, and Other Conveniences for 'ggplot2'
-Version: 0.1.2
+Version: 0.1.2.009
```

Comment:
What is the `009` indicating here? If following conventions for Semantic Versioning, we'd want to remove leading `0`s.

Reply:
Ha, so, that `.009` was an accident, but actually what I meant was `.9000`. In R, it's the standard representation of a development version of the package that sits between semantic releases. This is because it's extremely common for people to access development versions directly from GitHub, so putting that in the DESCRIPTION indicates it sits between any GitHub tagged releases. See [dplyr](https://github.com/tidyverse/dplyr/blob/main/DESCRIPTION) for example, and the [R Packages](https://r-pkgs.org/release.html) book section on releasing describing the process.

Reply:
Aha! Yes, that makes sense 😄